Jenkinsfile

```
@@ -128,7 +128,7 @@ pipeline {
                                 tools = 'mvn npm node yarn'
                             },
                             nuget: {
-                                agent = 'alpine-jdk8-mvn3.6-mono5-nuget3.4'
+                                agent = 'alpine-jdk8-mvn3.6-mono5-nuget5.7'
```

Thanks for changing this, but it requires to add a new Jenkins agent which we currently don't have and I noticed there is an issue with the authentication in nuget 5.7 (not related at all with your changes). Could you please switch this back to `nuget3.4` for now? Thanks in advance! :) 

```suggestion
                                agent = 'alpine-jdk8-mvn3.6-mono5-nuget5.7'
```

---

No worries, will do.